libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.ts

```diff
@@ -207,6 +211,20 @@ export class SkyDateRangePickerComponent
   @HostBinding('style.display')
   public display: string | undefined;
 
+  /**
+   * The template driven required input. We need to watch for this - but do not want to document it outside of the typical Angular template driven form documentation.
+   * @internal
+   */
+  @Input()
```

Comment:
I added the transform. We chatted some in flex - the transform here causes the value to always be a `boolean` type. I think it is more efficient to do this in the setter that will only fire on changes to `required` than having logic in `ngOnChanges` that would run every time change detection runs on the component. 

I did add an if-statement in the setter that ensures that the logic only runs on changes to `required`. This gives the same check that Angular was doing in `ngOnChanges` but ensures that it only runs when the setter is called instead of on every change detection cycle.